### PR TITLE
Exceptions

### PR DESCRIPTION
**Pull request by kambalas**

### CODE REVIEW

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 1]
```diff
@@ -45,14 +47,31 @@
...
 	}
 
 	private async Task Clicked(Posts post)
 	{
-		await DBservice.DeletePostAsync(post);
-		pageService.posts = await DBservice.GetMyPostsAsync(Email);
+		try
+		{
+		  await DBservice.DeletePostAsync(post);
+  		pageService.posts = await DBservice.GetMyPostsAsync(Email);
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:68]: formating

File: ridalot2.0/ridalot2.0/Pages/MyPosts.razor [hunk 2]
```diff
@@ -45,14 +47,31 @@
 
 	protected override async Task OnInitializedAsync()
 	{
-		pageService.posts = await DBservice.GetMyPostsAsync(Email);
-		pageService.img = await DBservice.GetImagesAsync();
+		try
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/MyPosts.razor:50]: formating

File: ridalot2.0/ridalot2.0/Pages/Feed.razor [hunk 3]
```diff
@@ -79,18 +79,44 @@
 
 	protected override async Task OnInitializedAsync()
 	{
-		pageService.posts = await DBservice.GetFeedPostsAsync();
-		pageService.img = await DBservice.GetImagesAsync();
+		try
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/Feed.razor:82]: formating

File: ridalot2.0/ridalot2.0/Pages/CreateAnAd.razor [hunk 4]
```diff
@@ -158,12 +158,20 @@ else
 	}
 	public void CalculatePayment()
 	{
-		pay = del(10.1f, floor);
+		try
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/CreateAnAd.razor:161]: formating

File: ridalot2.0/ridalot2.0/Shared/MainLayout.razor [hunk 5]
```diff
@@ -1,27 +1,35 @@
...
-		<div class="top-row px-4">
-			<LoginControl />
-			<a class="basic" href="profile">profile</a>
-		</div>
-		<!--Main Body-->
-		<article class="content px-4">
-			@Body
-		</article>
-	</main>
+    <main>
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Shared/MainLayout.razor:8]: where's the login control? style?

**Review by Astijq — APPROVED** [review]

**Review by neringamaj — APPROVED** [review]

**Pull request merged**